python/data/__init__.py

```diff
@@ -23,3 +25,13 @@ def items_from_content_file(datadir=None, filename="content.json.gz"):
 
     with gzip.open(full_filename, mode='rt') as content_file:
         yield from ijson.items(content_file, prefix='item')
+
+def write_content_file(filename, content, datadir=None):
+    if datadir is None:
+        datadir = os.getenv("DATADIR")
```

Comment:
Probably would have to add `datadir = os.getenv("DATADIR") or "data"` here to make sure there is a default if the `DATADIR` variable is not set.

Reply:
Good spot, I've added a default now :+1: